### PR TITLE
Added team page.Issue no(#7).

### PR DESCRIPTION
**Pull request by Anupam-dagar**

### CODE REVIEW

**Comment by gauthamzz** [comment]
could you post a live demo url

**Comment by Anupam-dagar** [comment]
Oh,i forgot to comment that.
Live link:
https://anupam-dagar.github.io/humblefoolcup/team.html

**Comment by gauthamzz** [comment]
@Anupam-dagar could we try a better cards. It doesnt look that good

**Comment by Anupam-dagar** [comment]
Ok,I will make them better.

File: team.html [hunk 1]
```diff
@@ -0,0 +1,362 @@
...
+          <li>
+            <a href="index.html">Sponsors</a>
+          </li>
+         <li>
+            <a href="index.html">Faq</a>
+          </li>
+          <li>
+            <a href="team.html">Team</a>
+          </li>
+
```
Comment (gauthamzz) [inline team.html:79]: please provide link for humble fool tribute page also over here

**Review by gauthamzz — CHANGES_REQUESTED** [review]

**Comment by magician03** [comment]
@Anupam-dagar , thanks for your work.

Your branch seems to have merge conflicts. Please rebase/ merge with current master. :)

**Comment by Anupam-dagar** [comment]
I am remaking it,header and footer are left.I will fix merge conflicts after it is ready.

**Pull request closed without merging**